takeaway-challenge.rb

```diff
@@ -0,0 +1,27 @@
+require './lib/dish'
+require './lib/menu'
+require './lib/menufactory'
+require './lib/orderfactory'
+require './lib/order'
+require './lib/invoice'
+require './twilio'
+
+
+list1 = [{:name=>:nigiri_sushi, :price=>13.99},
+              {:name=>:tonkotsu_ramen, :price=>9.95},
+              {:name=>:teriyaki_salmon, :price=>8.80}]
+
+list2 = [{:name=>:dead_hippy, :price=>8.99},
+              {:name=>:black_palace, :price=>9.95},
+              {:name=>:buffalo_chicken, :price=>8.80},
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)